database/migrations/2020_11_24_150537_create_skill_categories_skills_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateSkillCategoriesSkillsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('skill_categories_skills', function (Blueprint $table) {
```

Comment:
Accordion to Laravel [pivot table naming convention conenvtion](https://laravel.com/docs/8.x/eloquent-relationships#many-to-many-table-structure), this table should be called `skill_skill_category`.